third_party/move/move-compiler-v2/transactional-tests/tests/inlining/bug_10991_typed.exp

```diff
@@ -0,0 +1,41 @@
+comparison between v1 and v2 failed:
+- processed 2 tasks
+- 
+- task 0 'publish'. lines 1-10:
+- Error: error[E01013]: unsupported language construct
+-   ┌─ TEMPFILE:7:22
+-   │
+- 7 │         assert!(foo(|x: u64, _: u64| x, |_: u64, y: u64| y, 10, 100) == 110, 0);
+-   │                      ^^^^^^ Typed parameter to lambda is only allowed in Move 2 and beyond
```

Comment:
nit: could we reword this to - "Explicit type annotations for lambda parameters are only allowed ..."

Because the lambda parameters are indeed typed even in Move 1, just that explicit type annotations are not allowed.

Reply:
Done.